app/lib/app/App.js

```diff
@@ -42,6 +42,8 @@ export default function App({ secvisogramPage }) {
         .then(
           (result) => {
             setUserInfo(result)
+            // make sure the first call is a GET so that the CSRF cookie is set
+            api.backend.getAboutInfo().then(() => {})
```

Comment:
Return or await api promise here to track its progress.

Reply:
I now handle that in CsrfApiRequest and only make the call if no cookie is set.